src/app/app.routing.ts

```diff
@@ -56,7 +56,7 @@ export const allRoutes: Routes = [
   },
   { path: "login", component: LoginComponent },
   { path: "404", component: NotFoundComponent },
-
+  { path: 'entity-not-found', component: EntityNotFoundComponent },
```

Comment:
As you can see above (line 58) we already have a `NotFoundComponent`. I think we can reuse the same UI instead of introducing a similar component additionally. Probably you can just route to /404 to even reuse the same existing paths, @Saransh1524